specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -5189,6 +5203,48 @@
       },
       "description": "Response from a List SynonymMaps request. If successful, it includes the full definitions of all synonym maps."
     },
+    "EncryptionKey": {
+      "properties": {
+        "keyVaultKeyName": {
+          "type": "string",
+          "description": "The name of the KeyVault key."
+        },
+        "keyVaultKeyVersion": {
+          "type": "string",
+          "description": "The version of the KeyVault key."
+        },
+        "keyVaultUri": {
+          "type": "string",
+          "description": "The URI of the KeyVault."
+        },
+        "accessCredentials": {
+          "$ref": "#/definitions/AzureActiveDirectoryApplicationCredentials",
+          "description": "The credentials to use to authenticate to the provided KeyVault. Can be left unspecified if MSI (Managed Service Identity) is enabled for the service.",
+          "externalDocs": {
+            "url": "https://aka.ms/azure-search-msi"
+          }
+        }
+      },
+      "required": [
+        "keyVaultKeyName",
+        "keyVaultKeyVersion",
+        "keyVaultUri"
+      ],
+      "description": "The configuration to use encryption with customer managed keys."
+    },
+    "AzureActiveDirectoryApplicationCredentials": {
+      "properties": {
+        "applicationId": {
+          "type": "string",
+          "description": "The application id."
```

Comment:
Not helpful. AAD stuff is already confusing; Please enlighten readers as much as you can.